source/android/adaptivecards/src/main/java/io/adaptivecards/renderer/input/customcontrols/IValidatedInputView.java

```diff
@@ -0,0 +1,6 @@
+package io.adaptivecards.renderer.input.customcontrols;
+
+public interface IValidatedInputView
+{
```

Comment:
For new files please add the Microsoft copyright header that other platform source files already include.

Reply:
Fixed